build/proto/Dockerfile.arm64

```diff
@@ -0,0 +1,100 @@
+# Dockerfile for building .proto files.
```

Comment:
I think you could use setup-proto and that should allow you to clean this DOCKERFILE up a lot.

Reply:
Agree but am also OK with a ticketed follow-up to DRY it up, given folks are strongly desiring having this client out

Reply:
+1 for tidying this up afterwards.  I'd like to get us publishing the generated classes asap